components/Collection.vue

```diff
@@ -16,10 +16,10 @@
 
 		<slot name="before-description" v-bind="$props"></slot>
 
-		<section class="description" v-if="stac.description">
+		<section class="description" v-if="stac.description || stac.deprecated || supportedBy || affectedByMissing">
 			<h3>Description</h3>
 
-			<Description :description="stac.description"></Description>
+			<Description v-if="stac.description" :description="stac.description"></Description>
```

Comment:
The Description header is shown even if no description is shown, may also apply to other components.

Reply:
Yes, I think we can skip the description header in those cases. The proposed solution sounds reasonable.

Reply:
That's on the edge, both seems fine for me. Implement your preferred way ;-)

Reply:
Implemented in c68e39c050f70c6f58d2a4fb9a64c1288c43cbca 👍 
Processes always keep their heading due to the guaranteed signature content.
Item and UdfRuntime were also changed by me in the initial commit, but don't have a "Description" heading anyway, so not need to update them in this commit.